muxer_example.cpp

```diff
@@ -0,0 +1,12 @@
+
+#include <XmfMuxer.h>
+
+
+int main() {
+	auto muxer = XmfWebMMuxer_New();
```

Comment:
you can remove the small C++ executable from the project now that you've got the C# bindings and sample code in C#

Reply:
updated